src/util.cpp

```diff
@@ -538,8 +538,10 @@ CLicenceDlg::CLicenceDlg ( QWidget* parent ) : CBaseDlg ( parent )
 
     QVBoxLayout*  pLayout    = new QVBoxLayout ( this );
     QHBoxLayout*  pSubLayout = new QHBoxLayout;
-    QLabel*       lblLicence = new QLabel ( tr ( "This server requires you accept conditions before you can join. Please read these in the chat window." ), this );
-    QCheckBox*    chbAgree   = new QCheckBox ( tr ( "I have read the conditions and &agree." ), this );
+    QLabel*       lblLicence = new QLabel ( tr ( "This server requires you accept conditions before you can join." ) +
+                                            "<br/><br/>" +
+                                            tr ( "Please read these in the chat window." ), this );
```

Comment:
This splits the message in two, and will need the translations to be updated similarly. This kind of change could probably be done without understanding the translated language.

Reply:
It's more a matter of understanding the translation system...